io.github.suntorytimed.camunda-modeler.yaml

```diff
@@ -0,0 +1,50 @@
+app-id: io.github.suntorytimed.camunda-modeler
```

Comment:
the ID in PR title does not match the ID here.

Reply:
One commit was missing. Adjusted the ID to comply with the guidelines.

Reply:
Is upstream ok with the ID now? IMO the unverified badge and you can place a note saying that it is explicitly "not official" in metainfo should be enough.

Reply:
@nikku any thoughts on the proposed app ID of `io.github.camunda.camunda-modeler` and marking it as not official in the metainfo?